pandas/core/generic.py

```diff
@@ -3842,7 +3848,7 @@ def as_blocks(self, copy=True):
         .. deprecated:: 0.21.0
 
         NOTE: the dtypes of the blocks WILL BE PRESERVED HERE (unlike in
-              as_matrix)
+              as_array)
```

Comment:
leave this spelling, this is a top-level method name

Reply:
ok.